ERROR - 2022-02-20 09:32:56 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-02-20 09:32:56 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\laragon\www\ContactApp\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\laragon\www\ContactApp\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\laragon\www\ContactApp\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\laragon\www\ContactApp\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\laragon\www\ContactApp\app\Models\ContactModel.php(268): CodeIgniter\BaseModel->findAll()
#5 C:\laragon\www\ContactApp\app\Controllers\ContactController.php(46): App\Models\ContactModel->getContact()
#6 C:\laragon\www\ContactApp\system\CodeIgniter.php(825): App\Controllers\ContactController->tout_les_contacts()
#7 C:\laragon\www\ContactApp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ContactController))
#8 C:\laragon\www\ContactApp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\ContactApp\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
ERROR - 2022-02-20 09:33:00 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-02-20 09:33:00 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\laragon\www\ContactApp\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\laragon\www\ContactApp\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\laragon\www\ContactApp\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\laragon\www\ContactApp\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\laragon\www\ContactApp\app\Models\ContactModel.php(268): CodeIgniter\BaseModel->findAll()
#5 C:\laragon\www\ContactApp\app\Controllers\ContactController.php(57): App\Models\ContactModel->getContact()
#6 C:\laragon\www\ContactApp\system\CodeIgniter.php(825): App\Controllers\ContactController->recherche_contact()
#7 C:\laragon\www\ContactApp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ContactController))
#8 C:\laragon\www\ContactApp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\ContactApp\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
ERROR - 2022-02-20 09:33:05 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-02-20 09:33:05 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\laragon\www\ContactApp\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\laragon\www\ContactApp\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\laragon\www\ContactApp\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\laragon\www\ContactApp\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\laragon\www\ContactApp\app\Models\OrganisationModel.php(15): CodeIgniter\BaseModel->findAll()
#5 C:\laragon\www\ContactApp\app\Controllers\ContactController.php(113): App\Models\OrganisationModel->getOrganisation()
#6 C:\laragon\www\ContactApp\system\CodeIgniter.php(825): App\Controllers\ContactController->toute_les_organisations()
#7 C:\laragon\www\ContactApp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ContactController))
#8 C:\laragon\www\ContactApp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\ContactApp\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
ERROR - 2022-02-20 09:33:26 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2022-02-20 09:33:26 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
#0 C:\laragon\www\ContactApp\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\laragon\www\ContactApp\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\laragon\www\ContactApp\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#3 C:\laragon\www\ContactApp\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\laragon\www\ContactApp\app\Models\LoginModel.php(14): CodeIgniter\BaseModel->findAll()
#5 C:\laragon\www\ContactApp\app\Controllers\ContactController.php(142): App\Models\LoginModel->getLogin()
#6 C:\laragon\www\ContactApp\system\CodeIgniter.php(825): App\Controllers\ContactController->tout_les_login()
#7 C:\laragon\www\ContactApp\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ContactController))
#8 C:\laragon\www\ContactApp\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\ContactApp\public\index.php(37): CodeIgniter\CodeIgniter->run()
#10 {main}
